pkg/cortex/modules.go

```diff
@@ -611,7 +611,7 @@ func (t *Cortex) initMemberlistKV() (services.Service, error) {
 	t.Cfg.MemberlistKV.Codecs = []codec.Codec{
 		ring.GetCodec(),
 	}
-	t.MemberlistKV = memberlist.NewKVInitService(&t.Cfg.MemberlistKV)
+	t.MemberlistKV = memberlist.NewKVInitService(&t.Cfg.MemberlistKV, util.Logger)
```

Comment:
**suggestion [non-blocking]**: It may make sense to wrap the logger passed here with a `component` field?

Reply:
It sounds like a good idea, but log messages already look like this:

```
level=warn ts=2020-09-11T14:30:13.182589778Z caller=memberlist_client.go:541 msg="broadcast messages left in queue" count=1 nodes=1
```

So they include `memberlist_client.go`. Adding component to it would be probably too much.

Reply:
👍 Makes sense